cluster_management/src/main/java/com/pinterest/rocksplicator/CdcLeaderStandbyStateModelFactory.java

```diff
@@ -0,0 +1,75 @@
+package com.pinterest.rocksplicator;
+
+import org.apache.helix.NotificationContext;
+import org.apache.helix.model.Message;
+import org.apache.helix.participant.AbstractHelixLeaderStandbyStateModel;
+import org.apache.helix.participant.statemachine.StateModel;
+import org.apache.helix.participant.statemachine.StateModelFactory;
+import org.apache.helix.participant.statemachine.StateModelInfo;
+
+
+public class CdcLeaderStandbyStateModelFactory extends StateModelFactory<StateModel> {
```

Comment:
No state transitions from error state?

Reply:
Default implementations for the error state transitions are provided: https://github.com/apache/helix/blob/386a77d566f1dc0b480c3bcbdb4a2880a8b8a4a9/helix-core/src/main/java/org/apache/helix/participant/AbstractHelixLeaderStandbyStateModel.java
but I will add them here too for consistency